granary/bluesky.py

```diff
@@ -2247,24 +2259,30 @@ def preview_delete(self, at_uri):
   def base_object(self, obj):
     return base_object(obj)
 
-  def upload_media(self, media):
+  def upload_images(self, images):
```

Comment:
hmm, this removes support for eg videos. do we want to make this conditional on returned `Content-Type`? or add a `get_aspect_ratio` kwarg? or...?

Reply:
So the reason I changed this was so that we wouldn't try to use it for video once those are in (we don't currently support uploading video in Bluesky AFAICT from the code?). Happy to switch on Content-Type instead if you'd prefer that and you think it's trustworthy

Reply:
Ah, true! `upload_media` supported videos, but we currently only use it for images in `_create`. Yeah I think I'd like to keep that in `upload_media`, would you mind switching on `Content-Type`, and add another test for non-`image/...` uploads? Thank you!